dip-0006.md

```diff
@@ -138,11 +138,12 @@ Each phase lasts for multiple blocks (quorumDkgPhaseBlocks) to ensure that all m
 In this phase, the members of the new quorum are determined. This process is fully deterministic and results in exactly the same list seen by all members and observers.
 
 1. Retrieve the deterministic masternode list which is valid at quorumHeight
-2. Calculate `SHA256(SHA256(proTxHash, confirmedHash), SHA256(SHA256(llmqType, quorumHash)))` for each entry in the list
+2. If llmqType is the once for Platform use, keep only the HPMNs.
```

Comment:
We may want to simply use the masternode type in places like this. Otherwise we have to define HPMN in all docs.
```suggestion
2. If llmqType is the one for Platform use, keep only masternodes of type 1.
```